packages/alert/src/component.tsx

```diff
@@ -11,18 +11,21 @@ export function Alert({
   ...props
 }: PropsWithChildren<AlertProps>) {
 
+  const variantClasses = alert[type]
+  const iconVariantClasses = alert[`${type}Icon`]
```

Comment:
Not sure how react works, but if you change the prop from where it is used from let's say `negative` to `positive`... will this then be re-evaluated so that the classes will be updated to reflect the change? 🤔 🤷

Reply:
I removed any mentions of warning variant alerts in the site, and then updated the variant to "warning" on button click. Looks like updating the classes to use `warning`-variant tokens worked on runtime!
![variant-updating](https://user-images.githubusercontent.com/41303231/227919596-f362b35d-b2bf-498d-92d3-c85c99d7378b.gif)